tracer/src/Datadog.Trace/Sampling/SamplingMechanism.cs

```diff
@@ -0,0 +1,76 @@
+// <copyright file="SamplingMechanism.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace.Sampling;
+
+/// <summary>
+/// The mechanism used to make a trace sampling decision.
+/// </summary>
+internal static class SamplingMechanism
```

Comment:
Are these names/values finalized internally?

Reply:
The values are defined in an RFC. The name of the constants are internal to the dotnet tracer so we can change those.